src/chat/zulip/triagebot.md

```diff
@@ -0,0 +1,220 @@
+
+`@bot triage {high,medium,low,P-high,P-medium,P-low}`
+
+The configuration for this feature is:
+
+```toml
+[triage]
+remove = ["I-nominated"] # the set of labels to remove when this command is invoked
+high = "P-high"
+medium = "P-medium"
+low = "P-low"
+```
+
+## Applying labels to issues
+
+Anyone can apply a label to issues.
+
+TODO: any updates [after this PR merge](https://github.com/rust-lang/triagebot/issues/591)?
+
+The specific grammar can be found [here](https://github.com/rust-lang/triagebot/blob/master/parser/src/command/relabel.rs), but some examples are listed below. The grammar is intended to be fairly intuitive for people, to prevent needing to reach for documentation when using the bot.
+
+```
+@rustbot modify labels to +T-lang, -T-compiler
+```
+
+This will remove the `T-compiler` label and add the `T-lang` label. You can also omit the `+` sign, if you want, and it'll be implied.
+
+You can also write the same command in a few other ways:
+
+```
+@rustbot modify labels to +T-lang and -T-compiler
+@rustbot modify labels: +T-lang and -T-compiler
+@rustbot modify labels to +T-lang and -T-compiler
+```
+
+Note that the command can either terminate with a `.` or a newline, otherwise the bot will not parse the command successfully.
+
+### Errors
+
+The bot currently restricts the labels that can be applied by people outside the Rust teams. For example, they can't add the I-unsound label. Most of the time, you shouldn't hit this. Feel free to ping the release team if you feel that a label should be added to the set of allowed labels!
+
+### Enabling
+```toml
+[relabel]
+# any label is allowed to be set by team members (anyone on a team in rust-lang/team)
+# but these can be set by anyone in the world
+allow-unauthenticated = [
+    "C-*", # any C- prefixed label will be allowed for anyone
+           # independent of authorization with rust-lang/team
+    "!C-bug", # but not C-bug (order does not matter)
+]
+```
+
+## Requesting prioritization
+
+Users can request an issue to be prioritized by the Prioritization WG.
+
+To do so, you can invoke the following command:
+```text
+@rustbot prioritize
+```
+
+This will add the `I-prioritize` label to the issue, as well as creating a Zulip thread for the WG-prioritization members to discuss.
+
+### Errors
+The command fails if the issue has already been requested for prioritization (i.e. already has the `I-prioritize` label).
+
+### Enabling
+```toml
+[prioritize]
+# Name of the label used for requesting prioritization on issues
+label = "I-prioritize"
+# ID of the stream used by the Prioritization WG
+zulip_stream = 227806
+```
+
+## Major Changes
+
+A major change is an issue that will have a big impact on users. (TODO: improve description)
+
+The compiler team uses the major change process, which requires:
+ * An issue
+ * A second (TODO: replace term? The meaning could not be obvious to non-native English speakers)
```

Comment:
I could immediately understand what is a `second` in this context (probably technical ignorance on my side) but could it be perhaps reworded?